cli/flox/src/commands/generations/history.rs

```diff
@@ -56,7 +56,16 @@ impl Display for DisplayChange<'_> {
             Author:     {author}
             Host:       {host}
             Generation: {generation}
-            Summary:    {summary}"})
+            "})?;
+
+        if let Some(command) = &self.change.argv {
+            let command = command.join(" ");
+            write!(f, "{}", formatdoc! {"
+            Command:    {command}
+            "})?;
+        }
+        write!(f, "{}", formatdoc! {"
+        Summary:    {summary}"})
```

Comment:
I'm not sure if we want both Command and Summary, @zmitchell @djsauble any input?

Reply:
agree, i think we should show both
kind is describing "what" happend "argv"/"command" is describing "how" it happened, different thing imo

Reply:
I think it's redundant in certain cases
```
  Command:    flox install hello
  Summary:    installed package 'hello (hello)'
```
I don't think it's worth making it conditional on something like command being present and kind being an install at the moment, so 🚢